setup-conf.sh

```diff
@@ -36,6 +36,8 @@ checkpoint_timeout = ${CHECK_POINT_TIMEOUT}
 #archive_command = 'test ! -f ${WAL_ARCHIVE}/%f && cp -r %p ${WAL_ARCHIVE}/%f'
 EOF
 
+echo -e $EXTRA_CONF >> $CONF
+
```

Comment:
What are the default values of `EXTRA_CONF`? Maybe you need to define a default value here https://github.com/kartoza/docker-postgis/blob/develop/env-data.sh#L19

Reply:
If `EXTRA_CONF` is not defined, `echo -e $EXTRA_CONF` simply returns an empty line.

Reply:
But if you prefer, I can also add it to `env-data.sh`

Reply:
Either is ok, It was just for consistency sake.

Reply:
I added the default value in `env-data.sh`